packages/google_maps_flutter/google_maps_flutter_web/lib/src/marker_clustering_js_interop.dart

```diff
@@ -68,14 +68,14 @@ extension type MarkerClustererOptions._(JSObject _) implements JSObject {
 ///
 /// https://googlemaps.github.io/js-markerclusterer/classes/Cluster.html
 @JS('markerClusterer.Cluster')
-extension type MarkerClustererCluster._(JSObject _) implements JSObject {
+extension type MarkerClustererCluster<T>._(JSObject _) implements JSObject {
```

Comment:
can you `T extends JSObject`? That way we get a minimum amount of correctness on this type. Not as good as a union type, but it's something.